### ISSUE TITLE
When the code style is set to wrap long lines, comment sign is not properly inserted

### ISSUE THREAD

**Issue by tuscland** [ignatov/intellij-erlang#508]
Hi,

I have enabled the "Code Style" preference "Wrap when typing reaches right margin".

If I type a long comment (the lines starts with `%%`), when reaching the line width limit (100 characters), IDEA automatically wrap the text by inserting a new-line character. Unfortunately it also fails to insert the comment characters, which forces me to reformat the code.

I had a discussion with Andrey Dernov about the "Fill Paragraph" editor action, which does not work with Erlang mode (see https://youtrack.jetbrains.com/issue/IDEA-129190#comment=27-811804).

In summary: formatting comments within a constrained number of columns is not really working well at the moment. If possible please make it so that proper comment signs are inserted when using the code style option to wrap lines. Otherwise, supporting the "Fill Paragraph" editor action would be welcome.

Cam

**Comment by marco-m** [comment]
+1

**Comment by ignatov** [comment]
@marco-m Please provide more information, right now works [OK for me](https://www.dropbox.com/s/w4jm7w1ecfw3epv/erl-doc-wrap.mov?dl=0).

**Comment by marco-m** [comment]
@ignatov You are right on the automatic wrapping when reaching end of line, sorry for the confusion. I was referring to the "Fill Paragraph" feature. Maybe better to close this issue and I will open one specific for the fill paragraph editor action.

**Comment by ignatov** [comment]
For sure, please open a new issue about "Fill Paragraph" feature.
